install.sh

```diff
@@ -177,6 +177,26 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_bash_completion() {
+  NVM_PROFILE=$(nvm_detect_profile)
+  case "$NVM_PROFILE" in \
```

Comment:
My mistake - the `\` isn't needed here, but it is needed after each of the pipes below